SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -108,8 +108,9 @@ bool USpatialStatics::IsSpatialMultiWorkerEnabled(const UObject* WorldContextObj
 		{
 			FString OverrideMultiWorkerSettingsClass = SpatialGDKSettings->OverrideMultiWorkerSettingsClass.GetValue();
 			FSoftClassPath MultiWorkerSettingsSoftClassPath(OverrideMultiWorkerSettingsClass);
-			WorldSettings->MultiWorkerSettingsClass = MultiWorkerSettingsSoftClassPath.TryLoadClass<USpatialMultiWorkerSettings>();
-			checkf(WorldSettings->MultiWorkerSettingsClass != nullptr, TEXT("%s is not a valid class"), *OverrideMultiWorkerSettingsClass);
+			WorldSettings->GetMultiWorkerSettingsClass() = MultiWorkerSettingsSoftClassPath.TryLoadClass<USpatialMultiWorkerSettings>();
```

Comment:
Ah, this isn't modifying the default worker setting class anymore, just a local class that's returned. This will have to be reworked so that the override is respected whenever we attempt to call `GetMultiWorkerSettingsClass()`

Reply:
Wow I am suprised that this compiles.
I will fix the code and add a test for overriding this from the command line.

Reply:
This has been fixed and test cases added for overriding from the command line.